src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+// store.js
+
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Kindly ensure to also import `combineReducers` from redux, you might have forgotten it when importing `configureStore`.

Reply:
- [ ] Kindly ensure to also import `categoriesReducer` into your store file like you have done for your `booksReducer`.